libsql/src/replication/mod.rs

```diff
@@ -65,6 +65,19 @@ pub enum Frames {
     Snapshot(SnapshotFile),
 }
 
+/// Detailed logs about bytes synced with primary
+#[derive(Debug)]
+pub struct SyncUsageStats {
+    pub prefetched_bytes: u64,
+    pub prefetched_bytes_discarded_due_to_new_session: u64,
+    pub prefetched_bytes_discarded_due_to_consecutive_handshake: u64,
+    pub prefetched_bytes_discarded_due_to_invalid_frame_header: u64,
+    pub synced_bytes_discarded_due_to_invalid_frame_header: u64,
+    pub prefetched_bytes_used: u64,
+    pub synced_bytes_used: u64,
+    pub snapshot_bytes: u64,
```

Comment:
We should convert these to getters rather than public fields on a struct. More so for public structs that we expose to users and we should follow the style in the rest of the crate for that. If these are pub for internal reasons then they should be `pub(crate)` etc.

Reply:
Done.